datasource/opennms-direct/src/test/java/org/opennms/alec/datasource/opennms/jvm/DirectAlarmDataSourceTest.java

```diff
@@ -76,5 +78,30 @@ public void testNewOrUpdatedAlarmWhenAlarmIsASituation() {
         when(alarm1.getFirstEventTime()).thenReturn(new Date());
         when(alarm1.getLastEventTime()).thenReturn(new Date());
         dac.handleNewOrUpdatedAlarm(alarm1);
+        assertTrue(true);
```

Comment:
no sure for what we need that?

Reply:
Sonar was complaining about test method without assertion.

Reply:
ahhh good old sonar :-) maybe add a comment for why we have that assertion?